src/content/docs/style-guide/writing-strategies/install-docs-guidance.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Choose from our install methods
+topics:
+  - New Relic only
+  - Basic style guide
+  - Writing guidelines
+---
+
+Installing New Relic is a crucial piece in our customer's experience. A failure to install something not only prevents our customers from using New Relic, but it almost certainly guarantees they won't be our customers eventually. Whether it's installing APM agents, the infrastructure agent, sending us logs, or installing CodeStream, among other things.
+
+This doc is intended to help you decide where to send your reader when they want to install something.
+
+## Choose the best install method [#choose]
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        What's being installed?
+      </th>
+      <th>
+        Recommended installation path
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        * Infrastructure agent
+        * Logging
+        * APM agents (except C SDK, Go, PHP, and Ruby)
+        * One or more of these [on-host integrations](/docs/new-relic-one/install-configure/new-relic-guided-install-overview/#ohi-recipes)
+      </td>
+      <td>
+        Use [guided install](/docs/new-relic-one/install-configure/new-relic-guided-install-overview/).
+      </td>
+    </tr>
+    <tr>
+      <td>
+        * APM (especially C SDK, Go, PHP, and Ruby)
+        * Browser
+        * Synthetics
+        * Open telemetry
+        * Cloud and on-host monitoring integrations
+        * Logs
+        * Infrastructure
+        * Network performance monitoring
+        * Machine learning ops
+      </td>
+      <td>
+        Use the [Add more data](https://one.newrelic.com/nr1-core?state=cab9400b-31f1-fab7-cd28-db21bd312951) page.
+      </td>   
+    </tr>
+    <tr>
+      <td>
+        Anything not covered by guided install and add more data
+      </td>
+      <td>
+        Use a relevant docs install page.
+      </td>         
+    </tr>
+    </tbody>
+</table>
+
+## An overview of installation methods [#overview]
+
+Currently, we have three methods our customers can use to install New Relic and start monitoring something:
+
+1. Guided install
```

Comment:
I'd link from the list to each section below

Reply:
Perhaps if this doc were longer, I'd agree.